core/trino-main/src/main/java/io/trino/server/QueryExecutionFactoryModule.java

```diff
@@ -115,7 +115,8 @@
     @Override
     public void configure(Binder binder)
     {
-        var executionBinder = newMapBinder(binder, new TypeLiteral<Class<? extends Statement>>() {}, new TypeLiteral<QueryExecutionFactory<?>>() {});
+        MapBinder<Class<? extends Statement>, QueryExecutionFactory<?>> executionBinder =
+                newMapBinder(binder, new TypeLiteral<Class<? extends Statement>>() {}, new TypeLiteral<QueryExecutionFactory<?>>() {});
```

Comment:
nit: How about replacing types with `<>` here and L177?
```java
newMapBinder(binder, new TypeLiteral<>() {}, new TypeLiteral<>() {});
```

Reply:
the idea of `TypeLiteral<T>() {}` is to capture the `T` (hence anonymous subclass). It should be explicit.